sections/introduction.tex

```diff
@@ -0,0 +1,31 @@
+\section{Introduction}\label{sec:introduction}
+Datasets consist of entries where each entry contains a certain amount of meta-data fields which describes the data.
+Datasets are sometimes not used to their full potential, regarding the amount of information available within some datasets.
+There exist many feature engineering method, which capture different aspects of a dataset, and these might aid in solving a specific task.
+Meta-data is describes field within a data entry which and is usually used with supervised learning as labels for the algorithm used.
```

Comment:
```suggestion
Meta-data describes a field within a data entry which is usually used with supervised learning as labels for the algorithm used.
```
Jeg er ikke sikker på hvad dette skal betyde, men prøvede at rette.
Don't like this sentence :/